### PR TITLE
[BadgeBundle & ResultBundle] Badge bundle result rules

### LINKED ISSUE #998: Création de badge avec ressource résultat
Pouvoir attribuer automatiquement un badge selon les points obtenus dans la ressource résultat.

### PR DESCRIPTION
**Pull request by ptsavdar**
fixes #998

### CODE REVIEW

**Comment by Elorfin** [comment]
LGTM

File: plugin/result/Resources/translations/log.fr.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+log_resource-claroline_result-mark_added_filter: 'Attribuer une note'
+log_resource-claroline_result-mark_added_sentence: 'a attribué la note de <strong>%mark%</strong> à <strong>%receiver_user%</strong>'
```
Comment (stefk) [inline plugin/result/Resources/translations/log.fr.yml]: You can remove the "de" in this sentence.

Comment (ptsavdar) [inline plugin/result/Resources/translations/log.fr.yml]: Done. Thanks @stefk !

**Comment by stefk** [comment]
LGTM

**Pull request merged**